app/serializers/user_action_serializer.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+class UserActionSerializer
+  include JSONAPI::Serializer
+
+  attribute :user_action_event, :status, :subject_user_verification, :acting_ip_address,
+            :acting_user_agent, :created_at, :updated_at, :acting_user_verification
+
+  belongs_to :user_action_events
+  belongs_to :acting_user_verification
+  belongs_to :subject_user_verification
```

Comment:
These will append the user_verifications to the response which i don't think we want so these can probably be removed